packages/neon_framework/lib/src/widgets/dialog.dart

```diff
@@ -523,3 +555,395 @@ class NeonUnifiedPushDialog extends StatelessWidget {
         ],
       );
 }
+
+/// Shows an emoji picker.
+///
+/// When the user selects an emoji the dialog will pop and return the emoji.
+class NeonEmojiPickerDialog extends StatelessWidget {
```

Comment:
The emoji picker has an unconstrained hight.
This looks bad when the window is full screen

Reply:
According to https://m3.material.io/components/dialogs/specs there is no height constraint, but I agree that it doesn't look good. What do you think about setting the constraint in our dialog theme to twice the width? I think that is a reasonable limit that will still look good.

Reply:
Ideally it should be the same size as the NeonUserStatusDialog

Reply:
But it is generic and will be used in other places too (like the message input in Talk).

Reply:
shouldn't talk use a bottom modal that replaces the keyboard as many other messaging apps do?
It's not that important for now so feel free to use the suggested 1.5x factor.